src/core/src/bootstrap/Constants.py

```diff
@@ -323,3 +323,9 @@ class EnvLayer(EnumBackport):
         NOT_SELECTED: 6
     }
 
+    # Ubuntu Pro Client constants.
+    class UbuntuProClientSettings(EnumBackport):
+        FEATURE_ENABLED = True,
+        MINIMUM_PYTHON_VERSION_REQUIRED = (3, 5)
```

Comment:
Is this 3, 5 or 3.5? If 3.5, why a tuple?

Reply:
using this compare the python version. 
e.g., below:
`>>> t = (3,5)
>>> (2,5) < t
True
>>> (3,5) < t
False
>>> (3,4) < t
True
>>> (3,6) < t
False
`

Reply:
This looked cleaner to compare both minor and major versions.